### ISSUE TITLE
Error in create script: undefined method 'realpath'

### ISSUE THREAD

**Issue by paulbatum** [anaisbetts/starter-mobile#8]
<pre>
Pauls-MacBook-Pro-2:Projects pbatum$ git clone https://github.com/paulcbetts/starter-mobile.git
Cloning into 'starter-mobile'...
remote: Counting objects: 894, done.
remote: Compressing objects: 100% (495/495), done.
remote: Total 894 (delta 443), reused 829 (delta 382)
Receiving objects: 100% (894/894), 19.81 MiB | 1.09 MiB/s, done.
Resolving deltas: 100% (443/443), done.
Pauls-MacBook-Pro-2:Projects pbatum$ cd starter-mobile/
Pauls-MacBook-Pro-2:starter-mobile pbatum$ script/create StarterTestPb
script/create:34: undefined method `realpath' for File:Class (NoMethodError)
    from script/create:34:in `map'
    from script/create:34
</pre>

WAT

Maybe this is the problem?

<pre>
Pauls-MacBook-Pro-2:starter-mobile pbatum$ ruby -v
ruby 1.8.7 (2012-02-08 patchlevel 358) [universal-darwin12.0]
</pre>

**Comment by paulbatum** [comment]
Yeah this worked once I ran it with ruby 2.0. Maybe check the ruby version number at the start of your script?

**Comment by anaisbetts** [comment]
@paulbatum I'll probably just switch it with `Pathname` so it works everywhere. You gotta get on that Mavericks action :)

**Comment by paulbatum** [comment]
Ruby isn't a great option given that some Xamarin users run on Windows now.
Maybe you should stick node.exe in the repo and use that instead?

**Comment by anaisbetts** [comment]
Ugh. Sounds like a horrible life. People can install Ruby
